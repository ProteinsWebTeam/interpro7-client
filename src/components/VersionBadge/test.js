import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { VersionBadge } from '.';

const renderer = new ShallowRenderer();

describe('<VersionBadge />', () => {
  test('should render', () => {
    renderer.render(<VersionBadge version={'75.0'} />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
