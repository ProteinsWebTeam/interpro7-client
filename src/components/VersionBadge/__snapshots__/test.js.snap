// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VersionBadge /> should render 1`] = `
<svg
  height={30}
  width={30}
>
  <polygon
    fill="#783a58"
    points="30,15 22.5,2.0096189432334217 7.5,2.00961894323342 0,14.999999999999998 7.499999999999995,27.990381056766577 22.49999999999999,27.990381056766587 30,15.000000000000004"
  />
  <text
    fill="white"
    fontSize={12}
    fontWeight="bold"
    style={
      {
        "cursor": "default",
      }
    }
    textAnchor="middle"
    x={15}
    y={19.5}
  >
    75.0
  </text>
</svg>
`;
